FindGUROBI.cmake

```diff
@@ -18,13 +18,13 @@ if(EXISTS ${GUROBI_HOME} )
     string(STRIP ${GUROBI_VER} GUROBI_VER )
     message( "-- The retrieved version of Gurobi is: " ${GUROBI_VER} )
 
-    string(SUBSTRING ${GUROBI_VER} 0 2 GUROBI_VER_LIB)
+    string(SUBSTRING ${GUROBI_VER} 0 3 GUROBI_VER_LIB)
```

Comment:
It seems for GUROBI v11 there is no need to cut the last digit, i.e. `110` needs to stay `110`.